docs/add-to-project.md

```diff
@@ -0,0 +1,86 @@
+# Documentation for `add-to-project.yml` workflow
+
+[add-to-project.yml](../.github/workflows/add-to-project.yml) is a GitHub workflow which adds a repository's issues and/or PRs to a GitHub Project.
+
+It only works for [v2 projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects), not [classic projects](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/about-project-boards).
+
+## How to use
+
+To reuse the workflow, create a new workflow file in each repository whose tickets should get auto-added to your project.
+
+Example config for adding every newly opened issue:
+
+
+```yml
+name: Add repository issues to VERY-FINE-PROJECT 
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    uses: acdh-oeaw/prosnet-workflows/.github/workflows/add-to-project.yml@RELEASE_VERSION
+    secrets:
+      ADD_TO_PROJECT_TOKEN: ${{ secrets.GH_PROJECTS_TOKEN }}
```

Comment:
I would change `GH_PROJECT_TOKEN` to `ADD_TO_PROJECT_TOKEN`, as this is the variable name we use

Reply:
My reasoning behind using different names was (generally would be) to try to make variable names make _a bit more sense_ within their respective contexts.

So for someone looking at the workflow files, `ADD_TO_PROJECT_TOKEN` should be recognisable as a reference to a token _specific_ to this workflow. For someone looking at the repo's action variables and secrets, who's not been following any workflow-related developments, the name would be meaningless. They wouldn't necessarily be able to guess _what exactly_ `GH_PROJECT_*` is used for either, but it could at least be a clue (hence also shared prefix for token secret + project num var).

I think it was actually the variable name in the original workflow which made me feel action secrets/vars should be a bit more helpful :P (I didn't know and couldn't guess what PAT stood for, ended up googling for it).

But I can change the name so they read the same.